laws/shared/src/test/scala/algebra/laws/Rat.scala

```diff
@@ -47,14 +47,14 @@ class Rat(val num: BigInt, val den: BigInt) extends Serializable { lhs =>
     if (num == 0) throw new ArithmeticException("/0") else Rat(den, num)
 
   def /(rhs: Rat): Rat =
-    lhs * rhs.reciprocal
+    lhs * rhs.reciprocal()
 
   def **(k: Int): Rat =
     Rat(num.pow(k), den.pow(k))
 
   def toDouble(): Double = num.toDouble / den.toDouble
```

Comment:
Style wise, these pure functions shouldn't use () right?

Reply:
Right, it maybe make sens to remove the parentheses, I'm not sure if it will break mima